config/foundation_sdk.dev.yaml

```diff
@@ -35,6 +35,13 @@ inputs:
       transformations:
         - '%__config_dir%/compiler/alerting.yaml'
 
+  - openapi:
+      url: 'https://raw.githubusercontent.com/grafana/grafana/%grafana_version%/public/openapi3.json'
+      no_validate: true
+      package: dashboardmeta
```

Comment:
This would make more sense in the `dashboard` package, wouldn't it?

Reply:
I have conflicts if I use an existing package: 
```
Error: conflicting metadata: can not merge schemas
```